addons/mail/static/src/components/follower/follower.js

```diff
@@ -61,8 +61,12 @@ class Follower extends Component {
      * @private
      * @param {MouseEvent} ev
      */
-    _onClickRemove(ev) {
-        this.follower.remove();
+    async _onClickRemove(ev) {
+        ev.preventDefault();
+        ev.stopPropagation();
+        await this.follower.remove();
+        this.trigger('reload', { fieldNames:['message_follower_ids'], keepChanges: true });
+        this.trigger('o-hide-follower-list-menu');
```

Comment:
This looks like this is another problem?

I understand the "reload" part, but why suddenly decide to close the menu this way?

Reply:
@seb-odoo 
To trigger the `reload` event we need the component from which it will be triggered. 
By default there is a `_hide` method bound to `followerListMenu`.
But if we let the menu collapse before the `reload` event is triggered we won't be able to access the `follower` component to trigger the `reload` event.
That's why I needed to stop the click method to be handled by `followerListMenu`.
But according to it's current behaviour the menu should be collapsed after we remove a follower. 
So, I had to call the `_hide` method using a custom event.

Reply:
Ok thank you, I understand the issue a bit better.
Usually we try to resolve these issues with `isEventHandled`/`markEventHandled`.

Also note that what we do in master now is to always trigger the reload on the chatter record instead to avoid this kind of issue (`reloadParentView`).

Reply:
I have removed the use of `ev.stopPropogation` using `isEventHandled`/`markEventHandled`.
But we would still need the custom event to hide the `followerListMenu`